pkg/app/pagewriter/pagewriter_suite_test.go

```diff
@@ -8,6 +8,8 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+const requestID = "11111111-2222-4333-8444-555555555555"
```

Comment:
Giving this a more explicit name will help avoid shadowing of the variable somewhere unexpected. I could see us calling a variable `requestID` without thinking much more that `testRequestID`
```suggestion

const testRequestID = "11111111-2222-4333-8444-555555555555"
```